service/src/main/java/io/camunda/service/search/filter/UserTaskFilter.java

```diff
@@ -21,9 +21,9 @@ public final record UserTaskFilter(
     List<String> userTaskDefinitionIds,
     List<String> processNames,
     List<String> assignees,
-    List<String> userTaskState,
+    List<String> state,
```

Comment:
Maybe not related to this pull request, but I was wondering,  in `UserTaskFilter`, why we do have at the same time  `List<String> state`  and also 
```
    boolean created,
    boolean completed,
    boolean canceled,
```
isn't redundant ?